webserver/templates/user/import.html

```diff
@@ -55,6 +55,13 @@ <h2 class="page-title">{{ user.musicbrainz_id }}</h2>
         {{ user.auth_token }}
      </div>
 
+     <p><b> or </b></p>
+     <p>Import your listens from the lastfm backup file.</p>
```

Comment:
Please change the text to:

"Import your listens from a Last.fm backup file (.zip) or from a ListenBrainz export file (.json):"

And make it visually look more like the two sections above.